docs/website/11ty/resolve-path.transform.js

```diff
@@ -29,9 +30,20 @@ class PathResolver {
   static resolveLinks(dom, filePath) {
     dom.querySelectorAll('a[href^="."]').forEach((link) => {
       const resolved = PathResolver.resolveLink(link.href, filePath);
+
       if (resolved !== link.href) console.info(blue(`Rewrite link "${link.href}" to "${resolved}"`));
       link.href = resolved;
     });
+
+    const domain = new URL(url).hostname;
+    dom.querySelectorAll('a[href^="https"]').forEach((link) => {
```

Comment:
our link can also include `http` requests. maybe we can join 31 and this line? for example, you select all links, and then if `href` starts with `"."`, you execute the previous logic (with resolved), and if not, then we check for the domain